packages/terra-clinical-modal-manager/src/reducers/modalManager.js

```diff
@@ -0,0 +1,43 @@
+import { open, push, pop, maximize, minimize, defaultState } from './disclosureUtils';
+
+import {
+  OPEN,
+  CLOSE,
+  PUSH,
+  POP,
+  MAXIMIZE,
+  MINIMIZE,
+} from '../actions/modalManager';
+
+const supportedSizes = {
+  tiny: 'tiny',
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+  huge: 'huge',
+};
+
+const defaultModalState = Object.assign({}, defaultState, {
+  size: supportedSizes.small,
```

Comment:
This default state will be the initial state of the ModalManager when the application first mounts. There is no action at this point to consider. But it does beg the question whether or not a default is necessary at all...